SproxydKeysScan/index.js

```diff
@@ -27,6 +36,9 @@ Mandatory environment variables:
 Optional environment variables:
     RAFT_LOG_BEGIN_SEQ: offset to begin scanning from. Leave this out if you wish to begin 
     at latest cseq - LOOKBACK_WINDOW
+    LOG_LEVEL: defaults to info
+    DUMP_LEVEL: defaults to error
+    LOG_INTERVAL: logs summary of events in the given interval - defaults to 5 minutes
```

Comment:
where does this get defaulted to 5 minutes?

Reply:
env.js